Article/Article.js

```diff
@@ -1,29 +1,27 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.article.querySelector('.menu-button'); 
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.classList.text = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.panelButtons.addEventListener('hover', () => this.togglePanel());
```

Comment:
this should be on our expandButton and be a click event which calls the expandArticle function, not togglePanel